tests/system/View/TableTest.php

```diff
@@ -92,6 +97,58 @@ public function testSetFooting()
         );
     }
 
+    /**
+     * @depends testPrepArgs
+     */
+    public function testSetHeadingWithStyle()
+    {
+        // uses _prep_args internally, so we'll just do a quick
+        // check to verify that func_get_args and prep_args are
+        // being called.
```

Comment:
I don't get the comment. What do you want to say?

Reply:
Sorry, i am copy sample test related. `setHeading` and `setFooting` using `_prep_args`, Remove this comment?

Reply:
Or now change? comment move to above function?

Reply:
Do you understand what the comment says?

`_prep_args` should be `_prepArgs`, but I don't understand the sentence.
Where do we check?